client-runtime/src/main/java/com/microsoft/rest/v2/annotations/RequestContentType.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines the content type (MIME type) for the request body.
+ *
+ * If the Content-Type header present in {@link Headers} annotation then this annotation override it because this annotation is more specialized.
```

Comment:
"This annotation will override a Content-Type string in a {@link Headers} annotation."